File: src/main/java/cz/solutions/cockroach/PrintableEspp.java
package cz.solutions.cockroach;

import lombok.Value;

@Value
public class PrintableEspp {
    String date;
    int amount;
    String exchange;
    String onePricePurchaseDolarValue;
    String onePriceDolarValue;
    String oneProfitValue;
    String buyProfitValue;
    String buyCronePofitValue;
    int soldAmount;
    String taxableBuyCroneProfitValue;
}

File: src/main/kotlin/cz/solutions/cockroach/PrintableEspp.kt
package cz.solutions.cockroach

data class PrintableEspp(
    val symbol: String,
    val broker: String,
    val date: String,
    val amount: Double,
    val exchange: String,
    val onePricePurchaseDolarValue: String,
    val onePriceDolarValue: String,
    val oneProfitValue: String,
    val buyProfitValue: String,
    val buyCroneProfitValue: String,
    val soldAmount: Double,
    val taxableBuyCroneProfitValue: String
)